<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>437</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>289</width>
    <height>437</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>387</width>
    <height>437</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Horus Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../build-horusqt-Desktop_Qt_5_8_0_MinGW_32bit-Debug/horus.png</normaloff>../build-horusqt-Desktop_Qt_5_8_0_MinGW_32bit-Debug/horus.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-60</x>
     <y>390</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbUploads">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Upload Settings</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="cbUpload">
      <property name="text">
       <string>Upload Images/GIFs</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="cbOpenInBrowser">
      <property name="text">
       <string>Open in Browser</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="cbAskTitle">
      <property name="text">
       <string>Ask for Title</string>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="gbCopySettings">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>121</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Copy Settings</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QRadioButton" name="rbNoCopy">
      <property name="text">
       <string>None</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="rbCopyURL">
      <property name="text">
       <string>Copy URL</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="rbCopyImage">
      <property name="text">
       <string>Copy Image</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="gbServerSettings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Server Settings</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblServerURL">
      <property name="text">
       <string>Server URL</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="tbServerURL">
      <property name="placeholderText">
       <string>Server URL</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblServerPort">
      <property name="text">
       <string>Server Port</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="tbServerPort">
      <property name="placeholderText">
       <string>Server Port</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblAuthToken">
      <property name="text">
       <string>Auth Token</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="tbAuthToken">
      <property name="placeholderText">
       <string>Auth Token</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="cbSSL">
      <property name="text">
       <string>Use SSL</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnAbout">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbUploadBehavior">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>271</width>
     <height>58</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select how you want Horus to behave when you drop more than one file from your clipboard. Do you want it to zip all the files, and upload them at one URL, or upload them each individually, to their own, standalone URLs?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="title">
    <string>Multiple File Upload Behavior</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="rbUploadIndividual">
      <property name="text">
       <string>Upload Individually</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbUploadZip">
      <property name="text">
       <string>Upload as ZIP</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
